Google.py

```diff
@@ -55,8 +55,19 @@ def distance(self, googler_one, googler_two):
         Returns:
             distance: An integer representing their similarity or distance in the tree.
         """
-        #TODO: Implement function to determine distance between to Googlers.
-        return
+        if(googler_one == googler_two):
+            return 0
+        googler_one_managers = set()
+        googler_two_managers = set()
+        while(len(googler_one_managers.intersection(googler_two_managers)) == 0):
+            if(googler_one.manager_id != -1):
+                googler_one_managers.add(googler_one.manager_id)
```

Comment:
-1 indicates that the current googler is the "CEO" and has no manager so that the manager will only be added should they have one.

Reply:
Depends on how the googler_attribute_table.csv sets up the id to manager id relationship though.